src/main/java/com/nvidia/spark/rapids/jni/GpuTimeZoneDB.java

```diff
@@ -170,12 +174,56 @@ private void loadData(Executor executor) throws IllegalStateException {
     }
   }
 
+  /**
+   * load ZoneId.SHORT_IDS and append Z->UTC.
+   * The first 3 entries are: Z->UTC, PST->America/Los_Angeles, CTT->Asia/Shanghai
+   */
+  private void loadTimeZoneShortIDs() {
+    HostColumnVector.DataType type = new HostColumnVector.StructType(false,
+    new HostColumnVector.BasicType(false, DType.STRING),
+    new HostColumnVector.BasicType(false, DType.STRING));
+    ArrayList<HostColumnVector.StructData> data = new ArrayList<>();
+    // add Z->UTC
+    data.add(new HostColumnVector.StructData("Z", "UTC"));
+    // add PST CTT
+    for (Map.Entry<String, String> e : ZoneId.SHORT_IDS.entrySet()) {
+      if (e.getKey().equals("PST") || e.getKey().equals("CTT")) {
+        data.add(new HostColumnVector.StructData(e.getKey(), e.getValue()));
+      }
+    }
+    // add others
+    for (Map.Entry<String, String> e : ZoneId.SHORT_IDS.entrySet()) {
+      if (!(e.getKey().equals("PST") || e.getKey().equals("CTT"))) {
+        data.add(new HostColumnVector.StructData(e.getKey(), e.getValue()));
```

Comment:
Can we have a follow on issue to explore how to load this data faster? These APIs were set up for testing and even then they are not great. It would be a lot better to not have to create a new instance of a class for each item we want to work with.

Reply:
My original intention is: The short tz IDs have only small amount of items, it's fast to iterate them and PST/CTT have higher possibility.

I'll sort them and use binary search.
I tested, when the items has more than 10, binary search is fast than iteration.

Reply:
Done.